tasks/setup-docker-engine.yml

```diff
@@ -24,6 +24,12 @@
     name: "{{ docker_package_name }}{{ _docker_package_version | default('') }}"
     state: "{{ docker_package_state }}"
 
+- name: Explicitly install/Update Docker CLI too, on RedHat distros
```

Comment:
Looks good, one question though. Do we need to lock this, so we don't accidentally upgrade that to a newer version?

Reply:
Ah good point - I will try that out

Reply:
Yes, will lock that on our side